lib/hotel/menu/item.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Hotel.Menu.Item do
+  @moduledoc """
+  This module holds the changeset and schema to the Items table
+ """
+  use Hotel.Utils.Schema
+
+    alias Hotel.{
+    Menu.Item
+      }
+    schema "items" do
+      field :name, :string
+      field :description, :string
+      field :price, :string
+
+      many_to_many(
```

Comment:
So I read something about it being better to create explict tables for your many to many joins rather than using the macro it's much better for maintanability in the long run

Reply:
@zacck Please share the link.

Reply:
@zacck Okay, we can schedule a discussion session on discord.